rpa-manager/modules/ROOT/pages/processautomation-manage.adoc

```diff
@@ -262,3 +315,5 @@ Once it's ready, transition the project to the next phase of its life-cycle. You
 
 * xref:rpa-manager::myrpa-start.adoc[Starting Automation of Approved Processes]
 * xref:processautomation-prepare-project-category.adoc[Manage Process Categories]
+* xref:rpa-builder::activity-library-explorer.adoc[RPA Builder: Activity Library Explorer]
```

Comment:
Do you need the hard-coded link text here? If not, suggest removing them to enable auto updating if those headings change.

Reply:
Hi Hanna, I removed the ones, that were identical. Sometimes, I think, it's necessary to addimformation. In this case to clarify what categories, I added 'process' and to indicate that the link leads to another product's doc, I added 'Builder:'. Cris and I use this latter format consistently throughout the RPA docs. And because I like symmetry, I added link text for all links:)